frontend/apps/reader/modules/readerfont.lua

```diff
@@ -231,4 +258,26 @@ function ReaderFont:addToMainMenu(menu_items)
     }
 end
 
+function ReaderFont:onPinch()
+    local info = Notification:new{text = _("Changing font size")}
```

Comment:
Could you add a period?

So first it shows "Changing font size" and shortly thereafter "Font size set to %1."?
 https://github.com/koreader/koreader/blob/8b7e18a7d7d0e9422d0da31da0df4f157b383ae8/frontend/apps/reader/modules/readerfont.lua#L149